Screen.py

```diff
@@ -0,0 +1,206 @@
+        white_team = team_got_turn
+        black_team = team_doesnt_got_turn
+
+    white_team.update_score()
+    black_team.update_score()
+
+    start_team_score = 200
+    length = (white_team.score-2000) / 100
+    text = FONT.render("White team score:", False, colors.WHITE)
+    SCORE_BOARD.blit(text, (0, SCORE_BOARD.get_height() - 50))
+    pygame.draw.rect(SCORE_BOARD, colors.BLACK, (0, SCORE_BOARD.get_height() - 15, start_team_score, 10))
+    pygame.draw.rect(SCORE_BOARD, colors.WHITE, (0, SCORE_BOARD.get_height() - 15, length, 10))
+
+    length = (black_team.score-2000) / 100
+    x_pos = SCORE_BOARD.get_width() - start_team_score
+    text = FONT.render("Black team score:", False, colors.WHITE)
+    SCORE_BOARD.blit(text, (x_pos, SCORE_BOARD.get_height() - 50))
+    pygame.draw.rect(SCORE_BOARD, colors.WHITE, (x_pos, SCORE_BOARD.get_height() - 15, start_team_score, 10))
+    pygame.draw.rect(SCORE_BOARD, colors.BLACK, (x_pos, SCORE_BOARD.get_height() - 15, length, 10))
+
+
+def add_squares_to_board():
+    SCORE_BOARD.fill(colors.BROWN)
+    x = 0
+    y = HEIGHT_OF_SCOREBOARD
+    for line in range(BOARD_LINE):
+        tmp = line % 2
+        square_in_line = []
+        for tur in range(BOARD_LINE):
+            if tur % 2 == tmp:
+                color = colors.LIGHT_BROWN
+            else:
+                color = colors.DARK_BROWN
+
+            square_in_line.append(Square(x, y, color, tur, line))
+            x += Square.WIDTH
+        x = 0
+        y += Square.HEIGHT
+        squares.append(square_in_line)
+    pygame.display.flip()
+
+
+def color_all_square_to_original_color():
+    for line in squares:
+        for square in line:
+            if square.color != square.original_color:
+                square.coloring_square_by_original_color()
+
+
+MIDDLE_HORIZENTAL = SCREEN_WIDTH/2
+RECT_WIDTH = 200
+RECT_HEIGHT = 100
+
+
+def starting_screen():
+    screen.fill(colors.WHITE)
+    bg_image = pygame.image.load(os.path.join(PICTURES_PATH, 'opening_screen_picture.png'))
+    screen.blit(bg_image, (0, 0))
+    current_print_height = 150
+
+    one_player_rect = pygame.Rect(MIDDLE_HORIZENTAL - RECT_WIDTH/2, current_print_height, RECT_WIDTH, RECT_HEIGHT)
+    pygame.draw.rect(screen, colors.DARK_RED, one_player_rect)
+
+    text = FONT.render("One Player", False, colors.WHITE)
+    screen.blit(text, (one_player_rect.centerx - 50, one_player_rect.centery - 10))
+    current_print_height += 200
+
+    tow_player_rect = pygame.Rect(MIDDLE_HORIZENTAL - RECT_WIDTH/2, current_print_height, RECT_WIDTH, RECT_HEIGHT)
+    pygame.draw.rect(screen, colors.DARK_RED, tow_player_rect)
+
+    text = FONT.render("Tow Players", False, colors.WHITE)
+    screen.blit(text, (tow_player_rect.centerx - 50, tow_player_rect.centery - 10))
+    current_print_height += 100
+
+    pygame.display.flip()
+
+    while True:
+        for event in pygame.event.get():
+            if event.type == pygame.QUIT:
+                return
```

Comment:
Raise an exception instead of returning None.

Reply:
if I want  to write קבועים in a local function, where do I need to write them?
(look in line 167 - 169)